h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -108,6 +108,24 @@ public void initMetalearnerParams(Metalearner.Algorithm algo) {
     public Key<Frame>[] _base_model_predictions_keys; 
   }
 
+  @Override
+  public String[] adaptTestForTrain(Frame test, boolean expensive, boolean computeMetrics) {
+    return adaptTestForTrain(
+            test,
+            _output._origNames,
+            _output._origDomains,
+            _output._names,
+            _output._domains,
+            _output._metalearner._parms,
+            expensive,
+            computeMetrics,
+            _output.interactionBuilder(),
+            getToEigenVec(),
+            _toDelete,
+            false
+    );
```

Comment:
I could probably use as well:
```
    return _output._metalearner.adaptTestForTrain(test, expensive, computeMetrics);
```
trying it and checking the possible impact of that change

Reply:
actually, no, we can't use this simpler form, as the metalearner is trained on a totally different dataset...
even wondering if passing `_output._metalearner._parms` is such a good idea:
- `metalearner._parms` should be used to deal with the `level_one_frame` only, not the training/predictions sets.

Reply:
cancelled this approach as it was mixing 2 set of params (SE params and metalearner params) that should stay unrelated most of the time, especially when they're use to manipulate the input data.